comixed-release/src/main/assembly/scripts/run.sh

```diff
@@ -112,7 +112,8 @@ fi
 # build a list of JVM arguments
 
 if [[ $LIBDIR ]]; then
-  JVMOPTIONS="${JVMOPTIONS} -classpath ${LIBDIR}"
+  JVMOPTIONS="$JVMOPTIONS -classpath"
+  CLASSPATH="${LIBDIR}"
```

Comment:
See my comment in line 118 in the original source for the problem.  If `LIBDIR` exists, I made a copy of its value in `CLASSPATH`.

Note that if `LIBDIR` _doesn't_ exist:  `JVMOPTIONS` will be unchanged (i.e. won't contain `-classpath`), and the `CLASSPATH` will be an empty string in line 119 below.  (i.e. `-classpath $LIBDIR` won't be included)